main.py

```diff
@@ -77,6 +77,12 @@ async def on_chat(msg):
                 elif msg.text.lower().startswith("!version"):
                     console.log(f"Telling {user.get_username()} the current version")
                     await send_message(f"BrainBot v{__version__}", bot_chat)
+                elif msg.text.lower().startswith("!WhoisBrainBot?"):
```

Comment:
```suggestion
                elif msg.text.lower().startswith("!intro?"):
```

Reply:
Good idea, would you still need the question mark in that case?

Reply:
Fixed!